src/modules/CRUD/views/Map/index.js

```diff
@@ -6,29 +6,31 @@ import { connectCRUDProvider } from '../../services/CRUDProvider';
 
 import Map from './Map';
 
-export default connectAuthProvider(({
-  authenticated,
-  user,
-}) => {
-  const permissions = authenticated ? user.permissions : [];
-  return {
-    displayViewFeature: permissions.includes('terra.view_feature'),
-  };
-})(
-  connectCRUDProvider(
-    'getMapConfig',
-    'mapConfig',
-    'layersList',
-    'setMap',
-    'map',
-    'resizingMap',
-    'mapIsResizing',
-    'getFeaturesList',
-    'featuresList',
-    'feature',
-  )(
-    withRouter(
-      withNamespaces()(Map),
+export default withRouter(
+  connectAuthProvider(({
+    authenticated,
+    user,
+  }) => {
+    const permissions = authenticated ? user.permissions : [];
+    return {
+      displayViewFeature: permissions.includes('terra.view_feature'),
+    };
+  })(
+    connectCRUDProvider(
+      'getMapConfig',
+      'mapConfig',
+      'layersList',
+      'setMap',
+      'map',
+      'resizingMap',
+      'mapIsResizing',
+      'getFeaturesList',
+      'featuresList',
+      'feature',
+    )(
+      withNamespaces()(
```

Comment:
Not really sure that these line breaks improve readability..

```js
      withNamespaces()(Map),
```

Reply:
Done